code/menuui/techmenu.cpp

```diff
@@ -43,7 +43,7 @@
 
 #define NUM_BUTTONS	16
 #define NUM_TABS		3
-#define LIST_BUTTONS_MAX	42
+#define LIST_BUTTONS_MAX	50
```

Comment:
If there isn't a reason to keep the number low, perhaps we shouldn't stop at 50? A larger number might be more future proof.

Reply:
The most future-proof way would be to change the whole thing to a vector.  42 was sufficient for more than 20 years until a few mods started using smaller fonts.  50 will be sufficient unless the UI completely changes in such a way that the screen real estate vastly expands.  If that ever occurs, changing to a vector will most likely be part of it anyway.